owncloudApp/src/main/java/com/owncloud/android/authentication/AccountUtils.java

```diff
@@ -203,13 +203,6 @@ public static void updateAccountVersion(Context context) {
                                 accountMgr.getUserData(account, Constants.KEY_OC_VERSION)
                         );
 
-                        // copy cookies
-                        accountMgr.setUserData(
-                                newAccount,
-                                Constants.KEY_COOKIES,
-                                accountMgr.getUserData(account, Constants.KEY_COOKIES)
-                        );
-
```

Comment:
I know that we are going to handle this in a different way in upcoming PR, but i would keep it for the moment.